doc/user/methods/adaboost_regressor.md

```diff
@@ -0,0 +1,582 @@
+
+#### Constructor Parameters:
+
+| **name** | **type** | **description** | **default** |
+|----------|----------|-----------------|-------------|
+| `data` | [`arma::mat`](../matrices.md) | [Column-major](../matrices.md) training matrix. | _(N/A)_ |
+| `datasetInfo` | [`data::DatasetInfo`](../../tutorials/datasetmapper.md) | Dataset information, specifying type information for each dimension. | _(N/A)_ |
+| `responses` | [`arma::rowvec`]('../matrices.md') | Training responses (eg. values to predict).  Should have length `data.n_cols`.  | _(N/A)_ |
+| `numTrees` | `size_t` | Number of trees to train in the adaboost regressor. | `20` |
+| `minimumLeafSize` | `size_t` | Minimum number of points in each leaf node of each decision tree. | `10` |
+| `minimumGainSplit` | `double` | Minimum gain for a node to split in each decision tree. | `1e-7` |
+| `maximumDepth` | `size_t` | Maximum depth for each decision tree. (0 means no limit.) | `4` |
+
+### Training
+
+If traning is not done as part of the constructor call, it can be done with one 
+of the following versions of the `Train()` member function:
+
+  * `ab.Train(data, responses, numTrees=20, minimumLeafSize=10, minimumGainSplit=1e-7, maximumDepth=4)`
+     - Train on numerical-only data.
+
+---
+
+  * `ab.Train(data, datasetInfo, responses, numTrees=20, minimumLeafSize=10, minimumGainSplit=1e-7, maximumDepth=4)`
+     - Train on mixed categorical data.
+
+---
+
+Types of each argument are the same as in the table for constructors
+[above](#constructor-parameters).
+
+### Prediction
+
+Once a `AdaBoostRegressor` is trained, the `Predict()` member function can 
+be used to make class predictions for new data.
+
+ * `double predictedValue = ab.Predict(point)`
+   - ***(Single-point)***
+   - Predict and return the value for a single point.
+
+---
+
+ * `ab.Predict(data, predictions)`
+   - ***(Multi-point)***
+   - Predict and return values for every point in the given matrix `data`.
+   - The predictions for each point are stored in `predictions`, which is set to
+     length `data.n_cols`.
+   - The prediction for data point `i` can be accessed with `predictions[i]`.
+
+---
+
+#### Prediction Parameters:
+
+| **usage** | **name** | **type** | **description** |
+|-----------|----------|----------|-----------------|
+| _single-point_ | `point` | [`arma::vec`](../matrices.md) | Single point for prediction. |
+||||
+| _multi-point_ | `data` | [`arma::mat`](../matrices.md) | Set of [column-major](../matrices.md) points for prediction. |
+| _multi-point_ | `predictions` | [`arma::rowvec&`](../matrices.md) | Vector to store predictions into. |
+
+***Note:*** different types can be used for `data` and `point` (e.g.
+`arma::fmat`, `arma::sp_mat`, `arma::sp_vec`, etc.).  However, the element type
+that is used should be the same type that was used for training.
+
+### Other Functionality
+
+<!-- TODO: we should point directly to the documentation of those functions -->
```

Comment:
We can fix the `data::Save()` and `data::Load()` links now, take a look at the other `.md` documentation files for an example.